### PR TITLE
added fillColor for charts (only xlsx)

### PR DESCRIPTION
**Pull request by CrazyBite**
@PowerKiKi , I think my develop branch already rebuilded ..

### CODE REVIEW

**Comment by PowerKiKi** [comment]
It seems there are code style errors. You can fix them automatically with `./vendor/bin/php-cs-fixer fix`

File: src/PhpSpreadsheet/Chart/DataSeriesValues.php [hunk 1]
```diff
@@ -77,23 +77,35 @@ class DataSeriesValues
...
+=======
+     * Fill color.
+     *
+     * @var string
+     */
+    private $fillColor;
+
+    /**
+     * Create a new DataSeriesValues object
+>>>>>>> aea557bd70f6e4ac13ca625c342fce26ad544331
```
Comment (PowerKiKi) [inline src/PhpSpreadsheet/Chart/DataSeriesValues.php]: This kind of leftover from a not well resolved conflict should be removed

**Comment by CrazyBite** [comment]
@PowerKiKi already fixed, but it's wrong again ...

**Comment by PowerKiKi** [comment]
It seems you used a different PHP-Cs-Fixer with a different configuration than the one shipped with this project (`.php_cs.dist`). I think a3355c2 should be reverted, and PHP-Cs-Fixer should be used with PhpSpreadsheet custom configuration, meaning that the changes should be minimal.

**Comment by CrazyBite** [comment]
I hate travis :C

**Comment by PowerKiKi** [comment]
To avoid waiting on Travis every time, you can reproduce the check done by Travis locally with the following lines (saved in `check.sh`):

```sh
#!/bin/bash
set -xe

./vendor/bin/php-cs-fixer fix --dry-run --diff
./vendor/bin/phpcs --report-width=200 --report-summary  --report-full samples/ src/ tests/ --standard=PSR2 -n
./vendor/bin/phpunit
```

**Comment by CrazyBite** [comment]
There was 1 error:
1) PhpOffice\PhpSpreadsheetTests\SampleTest::testSample with data set "32 Chart read write" ('/home/travis/build/PHPOffice/...te.php')
PHPUnit_Framework_Exception: PHP Fatal error:  Call to a member function getFillColor() on boolean in /home/travis/build/PHPOffice/PhpSpreadsheet/src/PhpSpreadsheet/Writer/Xlsx/Chart.php on line 1102
Fatal error: Call to a member function getFillColor() on boolean in /home/travis/build/PHPOffice/PhpSpreadsheet/src/PhpSpreadsheet/Writer/Xlsx/Chart.php on line 1102
/home/travis/build/PHPOffice/PhpSpreadsheet/vendor/phpunit/phpunit/phpunit:52
ERRORS!
Tests: 4347, Assertions: 4720, Errors: 1, Skipped: 1, Incomplete: 97.

@PowerKiKi ,I don't understand this error, help me please

**Comment by PowerKiKi** [comment]
I fixed a few stuff and merged it, thanks.

Next time we'll have to cover new code with unit tests though...

**Pull request closed without merging**